composer.json

```diff
@@ -20,18 +20,19 @@
         "ext-zip": "*",
         "beberlei/doctrineextensions": "^1.2",
         "composer/package-versions-deprecated": "^1.8",
+        "doctrine/annotations": "1.11.2",
```

Comment:
Can be removed if this one is fixed: https://github.com/doctrine/annotations/issues/405